cts/cli/regression.tools.exp

```diff
@@ -4702,7 +4702,7 @@ Transition Summary:
   * Move       Public-IP                           ( cluster02 -> cluster01 )
   * Move       Email                               ( cluster02 -> cluster01 )
   * Stop       mysql-proxy:0                       (              cluster02 )  due to node availability
-  * Stop       promotable-rsc:0                    (     Promoted cluster02 )  due to node availability
+  * Stop       promotable-rsc:0                    (       Promoted cluster02 )  due to node availability
```

Comment:
This chunk probably doesn't really need to be committed, since this kind of stuff changes all the time.